### ISSUE TITLE
Exception for "Unknown" model

### ISSUE THREAD

**Issue by dsvoronin** [MarathonLabs/marathon#42]
marathon 0.1.1
seems like a valid case (it's always Unknown for emulator?)
```
Exception in thread "AndroidDevice(model=Unknown, serial=openstf-emulator.dev:1001)" java.lang.NumberFormatException: For input string: "unknown"
        at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
        at java.lang.Integer.parseInt(Integer.java:580)
        at java.lang.Integer.parseInt(Integer.java:615)
        at com.malinskiy.marathon.android.AndroidDevice.getDeviceFeatures(AndroidDevice.kt:43)
        at com.malinskiy.marathon.device.DeviceInfoKt.toDeviceInfo(DeviceInfo.kt:17)
        at com.malinskiy.marathon.android.executor.listeners.AnalyticsListenerKt.toTestResult(AnalyticsListener.kt:26)
        at com.malinskiy.marathon.android.executor.listeners.AnalyticsListener.handleTestRunResults(AnalyticsListener.kt:19)
        at com.malinskiy.marathon.android.executor.listeners.AbstractTestRunResultListener.testRunEnded(AbstractTestRunResultListener.kt:44)
        at com.malinskiy.marathon.android.executor.listeners.CompositeTestRunListener.testRunEnded(CompositeTestRunListener.kt:44)
        at com.android.ddmlib.testrunner.InstrumentationResultParser.handleTestRunFailed(InstrumentationResultParser.java:618)
        at com.android.ddmlib.testrunner.InstrumentationResultParser.submitCurrentKeyValue(InstrumentationResultParser.java:339)
        at com.android.ddmlib.testrunner.InstrumentationResultParser.parse(InstrumentationResultParser.java:310)
        at com.android.ddmlib.testrunner.InstrumentationResultParser.processNewLines(InstrumentationResultParser.java:267)
        at com.android.ddmlib.MultiLineReceiver.addOutput(MultiLineReceiver.java:103)
        at com.android.ddmlib.AdbHelper.executeRemoteCommand(AdbHelper.java:572)
        at com.android.ddmlib.AdbHelper.executeRemoteCommand(AdbHelper.java:383)
        at com.android.ddmlib.Device.executeShellCommand(Device.java:635)
        at com.android.ddmlib.testrunner.RemoteAndroidTestRunner.run(RemoteAndroidTestRunner.java:270)
        at com.android.ddmlib.testrunner.RemoteAndroidTestRunner.run(RemoteAndroidTestRunner.java:255)
        at com.malinskiy.marathon.android.executor.AndroidDeviceTestRunner.execute(AndroidDeviceTestRunner.kt:47)
        at com.malinskiy.marathon.android.AndroidDevice$execute$2.doResume(AndroidDevice.kt:89)
        at kotlin.coroutines.experimental.jvm.internal.CoroutineImpl.resume(CoroutineImpl.kt:54)
        at kotlinx.coroutines.experimental.DispatchTask.run(CoroutineDispatcher.kt:123)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:180)
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:293)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
```

**Comment by tagantroy** [comment]
Should be fixed in #45

**Comment by dsvoronin** [comment]
indeed
